ramls/request.json

```diff
@@ -70,17 +70,7 @@
     "item": {
       "description": "Copy of some item metadata (used for searching and sorting)",
       "type": "object",
-      "properties": {
-        "title": {
-          "description": "title of the item",
-          "type": "string"
-        },
-        "barcode": {
-          "description": "barcode of the item",
-          "type": "string"
-        }
-      },
-      "additionalProperties": false
+      "$ref": "request-itemdata.json"
```

Comment:
Why has this been extracting into a separate fragment?

Reply:
There was a name clash with the audit table 'item' property field: https://github.com/folio-org/raml-module-builder/pull/455
This has been resolved, now the audit table property field name needs to be configured in schema.json "auditingFieldName".
I moved request-itemdata.json back into request.json

Reply:
@julianladisch Thanks for sharing the background to this